### PR TITLE
Frontend/feat/react router prep

### PR DESCRIPTION
**Pull request by LionShareZoki**
Deleted 'app.tsx' as documentation recommends different approach to routing.
Prepared starting routes: '/', '/login', 'register'.

### CODE REVIEW

File: frontend/src/main.tsx
Comment (dariomrk) [inline frontend/src/main.tsx:1]: We can name this component `App` to denote that this is the root element of the app itself.

File: frontend/src/main.tsx [hunk 1]
```diff
@@ -2,9 +2,31 @@ import React from 'react'
 import ReactDOM from 'react-dom/client'
 import App from './App.tsx'
```
Comment (dariomrk) [inline frontend/src/main.tsx:3]: Accidental import?

File: frontend/src/main.tsx
Comment (LionShareZoki) [reply frontend/src/main.tsx:1]: Did that

File: frontend/src/main.tsx (hunk 1 above)
Comment (LionShareZoki) [reply frontend/src/main.tsx:3]: Deleted it

File: frontend/package.json [hunk 2]
```diff
@@ -10,8 +10,12 @@
     "preview": "vite preview"
   },
   "dependencies": {
+    "localforage": "^1.10.0",
```
Comment (dariomrk) [inline frontend/package.json:13]: Was `localforage` intentionally installed as an dependency?

File: frontend/package.json [hunk 3]
```diff
@@ -10,8 +10,12 @@
...
+    "match-sorter": "^6.3.1",
```
Comment (dariomrk) [inline frontend/package.json:14]: Was `match-sorter` intentionally installed as an dependency?

**Review by dariomrk — APPROVED** [review]

File: frontend/package.json (hunk 2 above)
Comment (LionShareZoki) [reply frontend/package.json:13]: It was possibly recommended to install with the router. Not sure.

File: frontend/package.json (hunk 3 above)
Comment (LionShareZoki) [reply frontend/package.json:14]: It was possibly recommended to install with the router. Not sure.

**Pull request merged**